installing/cluster-capabilities.adoc

```diff
@@ -0,0 +1,74 @@
+
+[NOTE]
+====
+Cluster administrators cannot disable a cluster capability after it is enabled.
+====
+
+include::modules/selecting-cluster-capabilities.adoc[leveloffset=+1]
+
+include::snippets/capabilities-table.adoc[]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/installing_aws/installing-aws-customizations.adoc#installing-aws-customizations[Installing a cluster on AWS with customizations]
+* xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-customizations[Installing a cluster on GCP with customizations]
+
+include::modules/explanation-of-capabilities.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../operators/operator-reference.adoc#cluster-operator-reference[Cluster Operators reference]
+
+include::modules/cluster-bare-metal-operator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Deploying installer-provisioned clusters on bare metal]
+* xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#preparing-to-install-on-bare-metal[Preparing for bare metal cluster installation]
+* xref:../post_installation_configuration/bare-metal-configuration.adoc#post-install-bare-metal-configuration[Bare metal configuration]
+
+include::modules/cluster-storage-operator.adoc[leveloffset=+2]
+
+include::modules/console-operator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../web_console/web-console-overview.adoc#web-console-overview[Web console overview]
+
+include::modules/cluster-csi-snapshot-controller-operator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots[CSI volume snapshots]
+
+include::modules/insights-operator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../support/remote_health_monitoring/using-insights-operator.adoc#using-insights-operator[Using Insights Operator]
+
+include::modules/operator-marketplace.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../operators/understanding/olm-rh-catalogs.adoc#olm-rh-catalogs[Red Hat-provided Operator catalogs]
+
+include::modules/cluster-samples-operator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../openshift_images/configuring-samples-operator.adoc#configuring-samples-operator[Configuring the Cluster Samples Operator]
+
+[role="_additional-resources"]
+[id="additional-resources_{context}"]
+== Additional resources
+* xref:../post_installation_configuration/enabling-cluster-capabilities.adoc[Enabling cluster capabilities after installation]
```

Comment:
The [xref must include an anchor ID](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references).